backend/src/bundles/payment/payment.service.ts

```diff
@@ -119,18 +119,66 @@ class PaymentService implements IPaymentService {
         const subscription: Stripe.Subscription =
             await this.createStripeSubscription(customer.id, priceId);
 
-        const { latest_invoice, id } = subscription;
+        const { latest_invoice, id, status } = subscription;
```

Comment:
Names of variables should be written in camelCase notation.

Reply:
from stripe